shared/provision/code-page/index.js

```diff
@@ -0,0 +1,479 @@
+// @flow
+import * as React from 'react'
```

Comment:
the entire kex flow. I kept it in one file so its all in one place but we could split it up if that helps. sometimes its easier if its in one file i think